### DISCUSSION TITLE
External link to an artifact in GitLab

### DISCUSSION THREAD

**Discussion by dimidevk** [jreleaser/jreleaser#653] [Q&A] [answered]
Hello dear @aalmiray!
In gitlab it is possible to create external links for the artifacts within the release. We upload our artifacts to nexus with the upload function of jreleaser. Is it possible to add this nexus-artifact-links as external urls to the release instead of uploading this files to gitlab? This is the only step we are doing manually now.

**Comment by aalmiray** [comment, marked as answer]
I suppose it's possible just have to configure the correct REST API call to make it happen. Do you have a an example of a public resource where I can I see what you expect to happen? If there's none then please paste an example release body or edited screenshot (to avoid leaking private data).  Thanks!

**Reply by dimidevk** [reply]
Great, it sounds cool! Unfortunately it is a private corporate repo, but i've tried to hide all the internal information with my mad gimp skills :)

This is the jreleaser.yml config:

![jreleaser_view](https://user-images.githubusercontent.com/52451715/147990314-cdb5edfc-f333-43de-8890-2c40226626e6.png)

And here we have the release editing form, where i've inserted the 2 external links manually, after they were uploaded to the artifactory:

![release_view](https://user-images.githubusercontent.com/52451715/147990398-4b9dbd86-4594-44ea-ae0a-c6bd305c7d6b.png)

My idea was, that it should be not to difficult to add the links to the release page automatically, since they are already being generated by nexus in the jreleaser upload step.
Please let me know, if you need something else and thank you very much for the support!

**Reply by aalmiray** [reply]
Thanks! I was not aware that you could use the Artifactory REST API to publish to Sonatype. Thought I had to add explicit support for it 😄 

Will have a look at GitLab's API for adding links to a release.

**Reply by dimidevk** [reply]
Great, thank you!

**Reply by aalmiray** [reply]
Filed and resolved https://github.com/jreleaser/jreleaser/issues/654
